gradle/libs.versions.toml

```diff
@@ -22,7 +22,7 @@ maven = "3.8.6"
 mavenResolver = "1.7.3"
 micrometer = "1.10.2"
 nessieBuildPlugins = "0.2.14"
-nessieClientVersion = "0.30.0"
+nessieClientVersion = "0.44.0"
```

Comment:
nit: could it be worth adding a comment here that this version is supposed to kept in sync with nessie in the used iceberg version?
i.e. https://github.com/apache/iceberg/blob/apache-iceberg-1.1.0/versions.props#L25